### ISSUE TITLE
How to get Latest story ?!!

### ISSUE THREAD

**Issue by erfankm7** [instagram4j/instagram4j#361]
in this version you can just see who has uploaded the latest story , but cant see story (Picture / Video ) url !!

my code is :
```
InstagramGetReelsTrayFeedResult feedResult = instagram.sendRequest(new InstagramGetReelsTrayFeedRequest());
      List<InstagramStoryTray> storyTrays = feedResult.getTray();
```

in the story trays you can just see username and some other information but no story link for picture or video !

**Comment by jemand771** [comment]
Did you open two issues for the same question? #356 
Stories are currently being reworked I think. #352

**Comment by erfankm7** [comment]
> Did you open two issues for the same question? #356
> Stories are currently being reworked I think. #352
@jemand771 
no , there are two kind of story : first of all is the latest story ( if you open Instagram app you see on top ) . second one , you search for public user (or private ) and you want see its story !

about #352 , after updating to v1.12 it doesnt work

**Comment by DanielePancottini** [comment]
@erfankm7 hi, you can use the following class to get story tray of a certain user: `InstagramGetUserReelMediaFeedRequest`

Ref to closed PR #333

**Comment by jvogit** [comment]
Addressed in [rewrite](https://github.com/instagram4j/instagram4j/tree/develop) or no longer relevant.
